Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportRenderer.h

```diff
@@ -91,6 +94,8 @@ namespace EMStudio
         const RenderOptions* m_renderOptions;
 
         const float DefaultFrustumDimension = 128.0f;
+        const float BoundMaxDistance = 150.0f;
+        const float TileSize = 1.0f;
```

Comment:
Nit: You might want to make them `static constexpr` and give them a `s_` prefix.